lab_6_pipeline/pipeline.py

```diff
@@ -112,12 +210,21 @@ def convert_morphological_tags(self, tags: str) -> str:  # type: ignore
         """
         Converts the Mystem tags into the UD format
         """
+        tags_list = re.split(r'\W+', re.match(r'[^|]+|', tags).group())
+        ud_tags = []
+        if tags_list[0] in ['PART', 'PR', 'CONJ', 'INTJ', 'COM', 'ADV', 'ADVPRO']:
```

Comment:
rework in a different way: it is easier to say WHAT we support instead of what we do NOT support.

also, it would be better to exactly specify what features we want to extract for each particular pos

Reply:
done